basebinaries/libjailbreak/libjailbreak.m

```diff
@@ -0,0 +1,82 @@
+//
+//  libjailbreak.m
+//  libjailbreak
+//
+//  Created by Jamie Bishop on 29/01/2018.
+//  Copyright © 2018 Electra Team. All rights reserved.
+//
+
+#import "libjailbreak.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netdb.h>
+
+#define BUFSIZE 1024
+#define HOSTNAME "127.0.0.1"
+
+#define JAILBREAKD_COMMAND_ENTITLE 1
+#define JAILBREAKD_COMMAND_PLATFORMIZE 2
+#define JAILBREAKD_COMMAND_FIXUP_SETUID 6
+
+// Global socket variable
+static int sockfd;
```

Comment:
apps are normally not going to make a ton of jbd calls so it's probably wise not to keep a global socket open

Reply:
Honestly I just didn’t like reopening it every time it ran